src/styles.scss

```diff
@@ -27,6 +27,10 @@ strong {
   font-weight: 700;
 }
 
+.secondary-btn {
+  background-color: $btn-color !important;
```

Comment:
Have you considered changing the value of `$secondary-color` instead of creating a new class?